package-lock.json

```diff
@@ -592,9 +592,9 @@
       }
     },
     "node_modules/@halcyontech/vscode-ibmi-types": {
-      "version": "2.11.0",
-      "resolved": "https://registry.npmjs.org/@halcyontech/vscode-ibmi-types/-/vscode-ibmi-types-2.11.0.tgz",
-      "integrity": "sha512-CCk/9rihuEeO3OK9FcHBtEplmOqX0wgYmnVHaLYdsWwmc7M4PYAf5gqwV2pzn0JJYW1LP/NX1oLc1vwKTm4x7g==",
+      "version": "2.12.1",
+      "resolved": "http://rdanart01plv.quarcad.net:4873/@halcyontech%2fvscode-ibmi-types/-/vscode-ibmi-types-2.12.1.tgz",
```

Comment:
Any reason why these are not pointing to `https://registry.npmjs.org/@halcyontech/vscode-ibmi-types/...`?

The build fails as you can see [here](https://github.com/IBM/vscode-ibmi-projectexplorer/actions/runs/10455694999/job/28951260970?pr=558) because of this:
```
npm error network request to http://rdanart01plv.quarcad.net:4873/@halcyontech%2fvscode-ibmi-types/-/vscode-ibmi-types-2.12.1.tgz failed, reason: getaddrinfo ENOTFOUND rdanart01plv.quarcad.net
```

Reply:
Using `npm i` after deleting the `package-lock.json` file on my side ended up fixing it. It was probably something in your npm config causing it to point to that private registry?

Reply:
Yes; it happened because I have Verdaccio here that acts as a proxy and cache for internal and external dependencies. So this will happen again if I run npm i from here.
Ideally, we should add a `.npmrc` file at the root of the repository to avoid this kind of issue, like this one: https://github.com/ARCAD-Software/arcad-ibmi-servers/blob/main/.npmrc